internal/commands/test_test.go

```diff
@@ -95,11 +96,19 @@ metadata:
 		t.Fatalf("could not process policy file: %s", err)
 	}
 
-	const expected = 2
-	actual := len(results.Failures)
-	if actual != expected {
-		t.Errorf("Multifile yaml test failure. Got %v failures, expected %v", actual, expected)
+	const expectedFailures = 2
+	actualFailures := len(results.Failures)
+	if actualFailures != expectedFailures {
+		t.Errorf("Multifile yaml test failure. Got %v failures, expected %v", actualFailures, expectedFailures)
 	}
+
+	const expectedSuccesses = 1
+	actualSuccesses := len(results.Successes)
+	if actualSuccesses != expectedSuccesses {
+		t.Errorf("Multifile yaml test failure. Got %v success, expected %v", actualSuccesses, expectedSuccesses)
+	}
+
+	fmt.Println(len(results.Successes))
```

Comment:
Lets just remove this println, seems like a debug that isnt needed anymore.

Reply:
@xchapter7x  Sorry, I have removed it. Thanks.